extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -236,6 +219,47 @@ export function updateOverflow_(content, measurer, maxHeight, fontSize) {
   });
 }
 
+/**
+ *
+ * @param {!Document} document
+ * @param {!Element} element
+ */
+export function buildDom(document, element) {
+  const content = document.createElement('div');
+  applyFillContent(content);
+  content.classList.add(CONTENT_CLASS);
+
+  const contentWrapper = document.createElement('div');
+  contentWrapper.classList.add(CONTENT_WRAPPER_CLASS);
+  content.appendChild(contentWrapper);
+
+  const measurer = document.createElement('div');
+  measurer.classList.add(MEASURER_CLASS);
+
+  realChildNodes(element).forEach((node) => contentWrapper.appendChild(node));
+  copyInnerHtml(contentWrapper, measurer);
+  element.appendChild(content);
+  element.appendChild(measurer);
+}
+
+/**
+ * @param {!Element} element
+ * @param {string} className
+ * @return {Element}
+ */
+function getDescendentByClass(element, className) {
+  return scopedQuerySelector(element, ` .${escapeCssSelectorIdent(className)}`);
+}
+
+/**
+ * Copies the inner html from src element to destination element
+ * @param {!Element} src
+ * @param {!Element} dst
+ */
+function copyInnerHtml(src, dst) {
+  dst./*OK*/ innerHTML = src./*OK*/ innerHTML;
```

Comment:
Not sure I understand why this is now a helper method.

Reply:
At a first glance, it's used twice, each case would require `/*OK*/` and possible breaking into two lines. I'm fine with this as-written

Reply:
It was so that I didn't need to repeat the `/*OK*/` in two places. Removed the helper

Reply:
Oof, I replied after Ryan's comment... 🤷

Reply:
Is this likely to be a commonly-used operation in `buildDom` for different elements?

Reply:
Can't answer definitively, but hopefully not. We likely also wouldn't want to hide innerHTML in a commonly used helper like this.